lib/rubocop/cop/rspec/shared_examples_style.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Enforces use of string or symbolic to titleize shared examples.
+      #
+      # @example EnforcedStyle: string (default)
+      #   # bad
+      #   it_behaves_like :foo_bar_baz
+      #   it_should_behave_like :foo_bar_baz
+      #   shared_examples :foo_bar_baz
+      #   shared_examples_for :foo_bar_baz
+      #   include_examples :foo_bar_baz
+      #
+      #   # good
+      #   it_behaves_like 'foo bar baz'
+      #   it_should_behave_like 'foo bar baz'
+      #   shared_examples 'foo bar baz'
+      #   shared_examples_for 'foo bar baz'
+      #   include_examples 'foo bar baz'
+      #
+      #  @example EnforcedStyle: symbolic
+      #   # bad
+      #   it_behaves_like 'foo bar baz'
+      #   it_should_behave_like 'foo bar baz'
+      #   shared_examples 'foo bar baz'
+      #   shared_examples_for 'foo bar baz'
+      #   include_examples 'foo bar baz'
+      #
+      #   # good
+      #   it_behaves_like :foo_bar_baz
+      #   it_should_behave_like :foo_bar_baz
+      #   shared_examples :foo_bar_baz
+      #   shared_examples_for :foo_bar_baz
+      #   include_examples :foo_bar_baz
+      #
+      class SharedExamplesStyle < Cop
+        include ConfigurableEnforcedStyle
+
+        def_node_matcher :shared_examples_style, <<-PATTERN
+          (send nil? {
+              :it_behaves_like :it_should_behave_like
+              :shared_examples :shared_examples_for
+              :include_examples
```

Comment:
I think we may have some useful constants available in the `RuboCop::RSpec::Language` namespace.